pkg/model/local_target.go

```diff
@@ -15,6 +15,10 @@ type LocalTarget struct {
 	ignores   []Dockerignore
 
 	repos []LocalGitRepo
+
+	// Indicates that we should force this to run in parallel with other local
```

Comment:
```suggestion
	// Indicates that we should allow this to run in parallel with other
```

1. maybe this highlights why "force_parallel" feels weird to me - what does that mean if it's the only target eligible to build?
2. the "other resources" are not limited to local

Reply:
done